static/locales/en-US.yaml

```
@@ -78,6 +78,8 @@ Search Filters:
 Subscriptions:
     # On Subscriptions Page
   Subscriptions: Subscriptions
+  # channels that were likely deleted
+  Error Channels: Error Channels
```

I feel like changing this to something like "channels with errors" feels a bit more descriptive.
```suggestion
  Error Channels: Channels with Errors
```

---

I wasn't really sure what to put for the text so i put Error Channels. I like your suggestion though